thorlcr/master/thgraphmanager.cpp

```diff
@@ -1134,7 +1136,9 @@ bool CJobManager::executeGraph(IConstWorkUnit &workunit, const char *graphName,
         cost_type cost = money2cost_type(calculateThorCost(nanoToMilli(graphTimeNs), numberOfMachines));
         if (cost)
             wu->setStatistic(queryStatisticsComponentType(), queryStatisticsComponentName(), SSTgraph, graphScope, StCostExecute, NULL, cost, 1, 0, StatsMergeReplace);
```

Comment:
will these graph CostExecute stats. be aggregated at the workflow level, and global level?

Reply:
The workflow level cost includes not only all the subgraph costs but also workflow engine cost.  There is special code to handle this calculation: when calculating workflow level cost, it adds the additional workflow engine cost to the subgraph cost aggregates.  Unfortunately, this also means that the graph aggregates cannot be calculated using the new method, as it would override workflow cost calculated with the special cost.

Reply:
I have looked into making use of the global stats collection object to handle CostExecute.  It may make the code cleaner.  However, this is not a trivial change and, probably, isn't worth doing in this PR.  (The workflow engine costs complicates things too much).